nodejs/awsx/aws_test.go

```diff
@@ -489,23 +507,23 @@ func validateAPITests(apiTests []apiTest) func(t *testing.T, stack integration.R
 
 			if tt.requiredToken != nil {
 				resp := GetHTTP(t, req, 401)
-				assertRequestBody(t, `{"message":"Unauthorized"}`, resp)
+				assertRequestBody(t, `{"message":"Unauthorized"}`, false, resp)
```

Comment:
can you do `false /*skipBodyValidation*/`?  Passing constants like this makes it hard to tell at his callsite what's going on.